src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.html

```diff
@@ -391,6 +391,7 @@ <h1 class="gct-heading h2">Gene Comparison Tool</h1>
                 <div *ngIf="genesTable.filteredValue?.length">
                   <button
                     class="pin-all-button"
+                    [disabled]="pinnedGenes.length >= maxPinnedGenes"
```

Comment:
Set the disabled property, rather than just a 'disabled' class, so that Playwright can confirm that the pin gene button [is disabled](https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-disabled) when pinned gene limit has been reached.